public/linkedin.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
the file formatting is really bad. You could use something like https://www.svgminify.com/ or other tool for minifing your svg file